### PR TITLE
Scale down API cluster as capacity not required

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

A reasonable provision for the API cluster.

- [ ] Run `terraform apply`.

### CODE REVIEW

**Comment by alexwlchan** [comment]
Better – but now we’re wasting a lot of capacity.

CPU:

```
a pair of c4.xlarge     2 * (1024 * 4)   = 8192

API tasks               2 * (1024 + 128) = 2304
Loris tasks             2 * (1024 + 128) = 2304
-----------------------------------------------
Spare                  8192 - (2304 * 2) = 3584
```

Memory:

```
a pair of c4.xlarge     2 * (1024 * 8)   = 16384

API tasks               2 * (2048 + 128) =  4352
Loris tasks             2 * (2048 + 128) =  4352
------------------------------------------------
Spare                 16384 - (4352 * 2) =  7680
```

tl;dr: If we’re paying for c4.xlarges, we can afford to run another API and another Loris task without paying an extra cent – deploys might be a bit slower, but the majority of the time is spent not-deploying.
